openedx/openedx-platform
# [BD-38] Update design of custom pages [BB-4819] [TNL-8543]

## Description

This PR updates the design of the "Pages" page in the studio platform to "Custom Pages" in this mockup https://www.figma.com/file/SeNpRff5HxzdpCgoD5pmlQ/Apps-and-resources-landing-page?node-id=5045%3A85855. In this new design only static tabs are displayed on the page.

## Supporting information

[TNL-8543](https://openedx.atlassian.net/browse/TNL-8543)
[BB-4819](https://tasks.opencraft.com/browse/BB-4819)

## Before
<img width="1368" alt="Screenshot 2021-10-05 at 06 40 51" src="https://user-images.githubusercontent.com/30666999/135966838-622fbc9c-b018-4dd1-a225-ca7078212bf7.png">

## After

<img width="1348" alt="Screenshot 2021-10-06 at 16 04 47" src="https://user-images.githubusercontent.com/30666999/136231342-13b178f8-3c98-4b80-abb3-9b8cdca6825c.png">

## Testing instructions

-  Checkout to this branch `tinumide/update_custom_pages_design` on edx-platform repository
- Spin up master devstack and visit  the Studio platform at `localhost:18010`
- Go to "Pages" from the dropdown of  the "Content" tab of a course
- Verify that the design has been updated to the mockup in the ticket [BB-4819](https://tasks.opencraft.com/browse/BB-4819)
